frontend/src/metabase/api/geojson.ts

```diff
@@ -0,0 +1,87 @@
+import type { Feature, FeatureCollection } from "geojson";
+import { t } from "ttag";
+
+import { computeMinimalBounds } from "metabase/visualizations/lib/mapping";
+import type { GeoJSONData } from "metabase-types/api";
+
+import { Api } from "./api";
+
+export const geojsonApi = Api.injectEndpoints({
+  endpoints: (builder) => ({
+    getGeoJSON: builder.query<GeoJSONData, string>({
+      query: (id) => ({
+        method: "GET",
+        url: `/api/setting/${id}`,
+      }),
+    }),
```

Comment:
is this used? I see why you might want this for the typing, but maybe it should be `/api/setting/geojson` without a param?